bin/beeline

```diff
@@ -23,8 +23,29 @@
 # Enter posix mode for bash
 set -o posix
 
+realpath () {
```

Comment:
Hm, is there no way to refactor these and source the function once? would be a good excuse to create such a common script here if it doesn't exist already.